source/install/pre_requirements.rst

```diff
@@ -146,18 +146,28 @@ With `QGIS-Plugin-Manager <https://pypi.org/project/qgis-plugin-manager/>`_ :
     You must install the ``Lizmap server`` plugin. The ``Lizmap`` plugin is designed only for QGIS desktop. Do not
     keep both on your server.
 
-.. warning::
-    For **security** reason, to enable the API on the QGIS server side, you must enable the environment variable
+For **security** reason, to enable the API on the QGIS server side, you must enable the environment variable
     ``QGIS_SERVER_LIZMAP_REVEAL_SETTINGS`` with the value set to ``True`` on QGIS server.
+    
+This variable will **expose** server settings such as QGIS server version, which is used by Lizmap Web Client.
+
+.. code-block:: ini
+
+    # Apache FCGI example 
+    FcgidInitialEnv QGIS_SERVER_LIZMAP_REVEAL_SETTINGS True
+    # nginx fastcgi 
+    fastcgi_param  QGIS_SERVER_LIZMAP_REVEAL_SETTINGS  1;
```

Comment:
```suggestion
    fastcgi_param  QGIS_SERVER_LIZMAP_REVEAL_SETTINGS  True;
```

I think it's more explicit that "1". All values will work 
https://github.com/3liz/qgis-lizmap-server-plugin/blob/master/lizmap_server/tools.py#L22

Reply:
i wondered how these values were managed :)